src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -1065,6 +1068,8 @@ private function createOrdersWithProduct(
         bool $isFulfilled = false,
     ): void {
         $customers = $this->generateCustomers($numberOfCustomers);
+
+        /** @var ProductVariantInterface $sampleProductVariant */
```

Comment:
If you will replace annotation with `Assert::notFalse($sampleProductVariant)`, will static analysis be still happy?

Reply:
We discussed this issue some time back and decided to convince static analysis with annotations rather than assertions